applications/asset_tracker_v2/src/ext_sensors/ext_sensors_bsec.c

```diff
@@ -115,6 +148,36 @@ static int settings_set(const char *key, size_t len_rd, settings_read_cb read_cb
 	return -ENOTSUP;
 }
 
+static void state_save(void)
+{
+	int ret;
+
+	ret = bsec_get_state(0, ctx.state_buffer, ARRAY_SIZE(ctx.state_buffer),
+				work_buffer, ARRAY_SIZE(work_buffer),
+				&ctx.state_buffer_len);
+
+	if (ret != BSEC_OK) {
+		LOG_ERR("bsec_get_state err: %d", ret);
```

Comment:
Is this scenario considered irrecoverable? Is using an assertion better here in case?

Reply:
Given what I can see in the API documentation, this should either always work or never work. I'll add an assert.